controllers/dbaas.redhat.com/configmap.go

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (C) 2020 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dbaasredhatcom
+
+import (
+	"context"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+)
+
+const (
+	PROVIDERDATAVALUE = "Crunchy Bridge managed PostgreSQL"
+	PROVIDERDATAKEY   = "provider"
+
+	INVENTORYDATAVALUE    = "CrunchyBridgeInventory"
+	INVENTORYKINDDATAKEY  = "inventory_kind"
+	CONNECTIONKINDDATAKEY = "connection_kind"
+	CONNECTIONDATAVALUE   = "CrunchyBridgeConnection"
+	CREDENTIALS_FIELDS    = "credentials_fields"
+	LOGO                  = "logo"
+	NAME                  = "crunchy-bridge-registration"
+	NAMESPACE             = "dbaas-operator" //configmap namespace
+	RELATEDTOLABELNAME    = "related-to"
+	RELATEDTOLABELVALUE   = "dbaas-operator"
+	TYPELABELNAME         = "type"
+	TYPELABELVALUE        = "dbaas-provider-registration"
+)
+
+var labels = map[string]string{RELATEDTOLABELNAME: RELATEDTOLABELVALUE, TYPELABELNAME: TYPELABELVALUE}
+
+func (r *CrunchyBridgeInventoryReconciler) CreateBridgeRegistrationConfigMap(mgr manager.Manager) error {
```

Comment:
Looking at the couple other controllers, seems the Crunchy folks like having a comment at the func level, let's play along and add one, too. We could probably move the bit about invoking prior to manager blah blah into a func comment here instead of on the new client/reader below?

Reply:
good catch added kubebuilder/RBAC in inventory controller

Reply:
It seems the receiver r for CrunchyBridgeInventoryReconciler is not actually used. What about changing it to a normal function without the receiver?

Reply:
updated